Makefile

```diff
@@ -57,7 +57,7 @@ unit-test-nocoverage:
 unit-test-nocoverage-host: golint-host govet-host
 	HOST_TEST=1 GOGC=1000 godep go test -v ./... -check.v
 
-build: golint
+build: golint govet
```

Comment:
It got balled up because of a squash, but it needs to be in there. Can 
we take a pass on this? I think this is a small enough change to not 
worry about it too much.

On 8 Dec 2015, at 13:25, unclejack wrote:

>> @@ -57,7 +57,7 @@ unit-test-nocoverage:
>> unit-test-nocoverage-host: golint-host govet-host
>> 	HOST_TEST=1 GOGC=1000 godep go test -v ./... -check.v
>>
>> -build: golint
>> +build: golint govet
>
> This seems to have sneaked in. Would you be OK with having this in a 
> separate PR which makes more changes to the makefile? I don't have a 
> strong opinion about this, but we should avoid this when possible.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/volplugin/pull/78/files#r47015868